cypress.json

```diff
@@ -3,12 +3,13 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "video": false,
-  "defaultCommandTimeout": 11000,
+  "defaultCommandTimeout": 21000,
   "pageLoadTimeout": 120000,
   "env": {
     "OKAPI_HOST": "folio-snapshot.dev.folio.org",
     "OKAPI_TENANT": "diku",
     "diku_login": "diku_admin",
-    "diku_password": "admin"
+    "diku_password": "admin",
+    "long_delay": 20000
```

Comment:
What is it? Where it can be usable? I can't find it in official docs:
https://docs.cypress.io/guides/references/configuration#Node-version

Reply:
you are right. It's piece of old code, will be removed from this PR

Reply:
removed